extension/threadpool/test/thread_parallel_test.cpp

```diff
@@ -11,13 +11,16 @@
 #include <array>
 #include <mutex>
 
-#include <executorch/extension/parallel/thread_parallel.h>
+#include <executorch/runtime/kernel/thread_parallel_interface.h>
 #include <executorch/runtime/platform/platform.h>
 
 using namespace ::testing;
 using ::executorch::extension::parallel_for;
 
-class ParallelTest : public ::testing::Test {
+#ifndef ET_USE_THREADPOOL
+#endif
```

Comment:
whoops. will update to remove after I finish going through the rest of the stack to avoid excess churn on the later diffs